datafusion/sqllogictest/test_files/parquet.slt

```diff
@@ -172,17 +172,17 @@ DROP TABLE test_table;
 # Setup alltypes_plain table:
 statement ok
 CREATE EXTERNAL TABLE alltypes_plain (
-  id INT NOT NULL,
-  bool_col BOOLEAN NOT NULL,
-  tinyint_col TINYINT NOT NULL,
-  smallint_col SMALLINT NOT NULL,
-  int_col INT NOT NULL,
-  bigint_col BIGINT NOT NULL,
-  float_col FLOAT NOT NULL,
-  double_col DOUBLE NOT NULL,
-  date_string_col BYTEA NOT NULL,
-  string_col VARCHAR NOT NULL,
-  timestamp_col TIMESTAMP NOT NULL,
```

Comment:
The columns in the file are actually nullable, even if in practice they don't have nulls. This fixes it but I'm not sure it's what we want. If the file says a column is nullable but the table schema says it's not... do we go with the table schema?